man/systemd-analyze.xml

```diff
@@ -758,6 +758,23 @@ Service b@0.service not loaded, b.socket cannot be started.
         operate on files inside the specified image path <replaceable>PATH</replaceable>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--security=<replaceable>BOOL</replaceable></option></term>
```

Comment:
Our thinking here was that the security verb currently appears mostly geared toward analyzing the security of the host it's ran on - while the verify verb is more commonly use to check an individual (or set) of units offline, so this fit better there, with the added benefit of doing one call (but this is not hugely important).
But it's arbitrary choices, so if the preference is to update the security verb, that's fine. We'll have to add support for --image/--root to it as well, since that covers only verify right now.